teams/libs/charter.md

```diff
@@ -0,0 +1,111 @@
+# Libs team Charter
+
+# Mission
+
+Managing the Rust standard library and official rust-lang crates: the review process (incl. guidelines), triaging issues, overseeing the implementation and documentation, and keeping the libraries maintainable and healthy. And overseeing the other two teams.
+
+## How does this further the Rust Project goals?
+
+TODO
+
+## Responsibilities & Commitments
+
+For the purposes of this document, the "**standard libraries**" refer to the libaries shipped with the rust compiler like `core`, `alloc`, `std`, `test`, `backtrace`, `panic_*`, etc, and "**official rust-lang libraries**" refer to other published rust-lang libraries that are intended to be used by others, like `compiler-builtins`, `libc`, `regex`, `socket2`, etc.
```

Comment:
Once https://github.com/rust-lang/rfcs/pull/3119 lands we can be much more precise about this list